config/application.rb

```diff
@@ -1,6 +1,32 @@
 require_relative 'boot'
 
-require 'rails/all'
+# frozen_string_literal: true
```

Comment:
this must be first line

Reply:
Otherwise looks great, thanks!

Reply:
also `require_relative 'boot'` is repeated, so I think removing the first two lines will do it.

Reply:
Fixed!